sql/updates/latest-dev.sql

```diff
@@ -0,0 +1,30 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+
+-- Update the functions for aggregation pushdown from partialize_finalize.sql
+-- to be parallel safe.
+
+CREATE OR REPLACE FUNCTION _timescaledb_internal.partialize_agg(arg ANYELEMENT)
+RETURNS BYTEA AS '@MODULE_PATHNAME@', 'ts_partialize_agg' LANGUAGE C STABLE PARALLEL SAFE;
+
+CREATE OR REPLACE FUNCTION _timescaledb_internal.finalize_agg_sfunc(
+tstate internal, aggfn TEXT, inner_agg_collation_schema NAME, inner_agg_collation_name NAME, inner_agg_input_types NAME[][], inner_agg_serialized_state BYTEA, return_type_dummy_val ANYELEMENT)
+RETURNS internal
+AS '@MODULE_PATHNAME@', 'ts_finalize_agg_sfunc'
+LANGUAGE C IMMUTABLE PARALLEL SAFE;
+
+CREATE OR REPLACE FUNCTION _timescaledb_internal.finalize_agg_ffunc(
+tstate internal, aggfn TEXT, inner_agg_collation_schema NAME, inner_agg_collation_name NAME, inner_agg_input_types NAME[][], inner_agg_serialized_state BYTEA, return_type_dummy_val ANYELEMENT)
+RETURNS anyelement
+AS '@MODULE_PATHNAME@', 'ts_finalize_agg_ffunc'
+LANGUAGE C IMMUTABLE PARALLEL SAFE;
+
+CREATE OR REPLACE AGGREGATE _timescaledb_internal.finalize_agg(agg_name TEXT,  inner_agg_collation_schema NAME,  inner_agg_collation_name NAME, inner_agg_input_types NAME[][], inner_agg_serialized_state BYTEA, return_type_dummy_val anyelement) (
+    SFUNC = _timescaledb_internal.finalize_agg_sfunc,
+    STYPE = internal,
+    FINALFUNC = _timescaledb_internal.finalize_agg_ffunc,
+    FINALFUNC_EXTRA,
+    PARALLEL = SAFE
+);
```

Comment:
```suggestion
```
This is not needed as CREATE OR REPLACE for all functions is appended to any update script we generate. You only need  this if you change signature for example.

Reply:
Ah, right, removed.